src/mscorlib/src/System/Diagnostics/DebuggerAttributes.cs

```diff
@@ -22,239 +21,6 @@ public sealed class DebuggerStepThroughAttribute : Attribute
     {
         public DebuggerStepThroughAttribute() { }
```

Comment:
I would move all of them, and add `[System.Runtime.CompilerServices.DependencyReductionRoot]` under `#if !PROJECTN` ifdef.

Reply:
Is `PROJECTN` coreclr?

Reply:
PROJECTN means .NET Native. This attribute is only used by the (closed source) .NET Native toolchain.

Reply:
So `#if PROJECTN` rather than `#if !PROJECTN`?

Reply:
Right.